.vscode/extensions.json

```diff
@@ -0,0 +1,6 @@
+{
+    "recommendations": [
+        "mflo999.lintel", 
```

Comment:
Can you remove that from recommendations? As the configuration helper is not something everyone should have while working on the project, so I guess it's not something that should live in a recommendations section 😊